sematic/ui/packages/react-slot-fill/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import Fill from "./Fill";
```

Comment:
I am not the author of the code under `react-slot-fill`, I only revised the code to fix issues.